### PR TITLE
update-vendor.sh: call update-go-workspaces.sh automatically

### PR DESCRIPTION
**Pull request by BenTheElder**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

In all cases when you update vendor you also need to update go-workspaces.sh

We automatically call it now, as a stopgap to make working with vendor easier until we can land a larger cleanup like https://github.com/kubernetes/kubernetes/pull/123627

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

/sig architecture
/area code-organization

### CODE REVIEW

**Comment by BenTheElder** [comment]
> An error has occurred. Please see more details in /tmp/update-vendor.DLcp/update-vendor.log
> Removing /tmp/verify-vendor.sh.HfrPcW

Will have to debug this locally in a bit ...
/hold

File: hack/update-vendor.sh [hunk 1]
```diff
@@ -431,3 +431,6 @@ reviewers:
 __EOF__
 
 kube::log::status "NOTE: don't forget to handle vendor/* and LICENSE/* files that were added or removed" >&11
+
+# TODO simplify update-vendor and eliminate this
+hack/update-go-workspaces.sh
```
Comment (liggitt) [inline hack/update-vendor.sh:436]: nit, move this above the final output and log it is happening (just like we log and call out to update-vendor-licenses)

**Review by liggitt — COMMENTED** [review]
one nit, then lgtm

**Comment by BenTheElder** [comment]
There's an error in CI but I don't think the logs persisted:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/126852/pull-kubernetes-dependencies/1826299960539222016
> An error has occurred. Please see more details in /tmp/update-vendor.DLcp/update-vendor.log
> Removing /tmp/verify-vendor.sh.HfrPcW

Will have to debug locally

**Comment by liggitt** [comment]
verify-dependencies calls update-vendor from inside a copied subtree... update-workspace might be unhappy being invoked in that subtree... try running verify-vendor locally

**Pull request closed without merging**